assets/chezmoi.io/mkdocs.yml

```diff
@@ -225,7 +225,9 @@ nav:
       - reference/templates/github-functions/index.md
       - gitHubKeys: reference/templates/github-functions/gitHubKeys.md
       - gitHubLatestRelease: reference/templates/github-functions/gitHubLatestRelease.md
+      - gitHubRelease: reference/templates/github-functions/gitHubRelease.md
```

Comment:
The files `gitHubRelease.md` and `gitHubReleaseAssetURL.md` are missing from this PR.

Reply:
Should be fine now.